src/engine/strat_engine/backstore/devices.rs

```diff
@@ -247,6 +248,32 @@ pub fn process_devices(
     Ok(infos)
 }
 
+enum MaybeEncrypted {
+    Encrypted(CryptHandle),
+    Unencrypted(PathBuf, PoolUuid),
+}
+
+impl MaybeEncrypted {
+    fn to_blockdev_path(&self) -> StratisResult<BlockDevPath> {
```

Comment:
I will document this. The idea was to get all `LibcryptErr` return types consolidated to the crypt module as I want to keep everything from `libcryptsetup-rs` in one place to avoid the need for re-exports from the crypt module or imports from libcryptsetup-rs outside of the crypt module. This allows us to cleanly depend on the crate in one place.

Reply:
Resolved.